CHANGELOG.md

```
@@ -9,42 +9,33 @@
 
 # [0.1.0](https://github.com/VilnaCRM-Org/website/compare/v1.6.0...v0.1.0) (2025-06-04)
 
-
 ### Features
 
-* **#29:** add swagger page ([#195](https://github.com/VilnaCRM-Org/website/issues/195)) ([ccebf1a](https://github.com/VilnaCRM-Org/website/commit/ccebf1a8e835b2ba2a7b02a3ae379f14b2f5815d)), closes [#29](https://github.com/VilnaCRM-Org/website/issues/29)
-
-
+- **#29:** add swagger page ([#195](https://github.com/VilnaCRM-Org/website/issues/195)) ([ccebf1a](https://github.com/VilnaCRM-Org/website/commit/ccebf1a8e835b2ba2a7b02a3ae379f14b2f5815d)), closes [#29](https://github.com/VilnaCRM-Org/website/issues/29)
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)